docs/contributing/docs-templates.md

```diff
@@ -187,29 +194,74 @@ Assume the reader has basic programming knowledge like the command line, code ed
 
 ---
 
-## Tutorial
+## Recipes
 
-### Near perfect example of a tutorial
+[Docs Recipes](/docs/recipes/) should act as discoverable, concise instructions for completing common Gatsby tasks without having to navigate elsewhere. A recipe should list requirements and a few short steps to complete a task, listing actionable instructions inline and omitting everything else. Recipes are smaller units than tutorials: multiple recipes could be linked from a reference guide or tutorial.
 
-[Main Gatsby tutorial](https://www.gatsbyjs.org/tutorial/)
+The components of a recipe are:
 
-### What should a tutorial be about?
+- The name of the recipe, which should describe a single task
+- Requirements and prerequisites
+- Step-by-step directions
+- A link to a working example
+
+Recipes should be short. If you're finding a recipe is becoming too long to fit on the Docs Recipes page due to including many prerequisites or steps, consider writing a tutorial instead.
 
-We need tutorials to explain how to perform a series of related tasks with Gatsby.
+### Recipe template
 
-### When to write a guide vs. a tutorial?
+````markdown
```

Comment:
??

Reply:
I'm not sure what the ?? are for, exactly!

Recipe requirements and a recipe template were added here in between reference guides and tutorials, so that's what this Markdown example is about. I thought a template with actual content would also do the work of showing an example, but I'm open to suggestions!